app/src/main/res/values/strings.xml

```diff
@@ -168,7 +168,7 @@
     <string name="prefs_app_location_title">Quran Data Directory</string>
     <string name="prefs_app_location_summary">Choose where to store Quran files.</string>
     <string name="prefs_sdcard_internal">Internal Storage</string>
-    <string name="prefs_sdcard_external">External SD Card</string>
+    <string name="prefs_sdcard_external">External SD Card %1$d</string>
```

Comment:
either need to change it for other languages as well or remove the string from all non-English languages so people can add them again.